LINQ/Task1.Tests/LinqTaskTests.cs

```diff
@@ -0,0 +1,285 @@
+using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+using System.Linq;
+using Task1.DoNotChange;
+
+namespace Task1.Tests
+{
+    [TestFixture]
+    public class Tests
+    {
+        [TestCase(6250, ExpectedResult = 0)]
+        [TestCase(0, ExpectedResult = 6)]
+        [TestCase(-1, ExpectedResult = 10)]
+        [TestCase(1, ExpectedResult = 5)]
+        public int Linq1_Limit_ReturnsCustomersCount(decimal limit)
+        {
+            return LinqTask.Linq1(DataSource.Customers, limit).Count();
+        }
+
+        [Test]
+        public void Linq1_NullSource_ThrowsArgumentNullException()
+        {
+            Assert.That(() => LinqTask.Linq1(null, 42).ToList(), Throws.ArgumentNullException);
+        }
+
+        [Test]
+        public void Linq2_CustomersAndSuppliers_2CustomersHaveSuppliers()
```

Comment:
The test method name could be more readable. The test's name should be more self-explained. For example;
MethodName_ShouldDoThat_WhenTheConditionIsMet.

However, I recommend you follow the 3A pattern when writing the test;
//Arrange
//Act
//Assert

For more info read the article: https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80